db/agents.yml

```diff
@@ -299,10 +327,18 @@ agents:
     name: RSSRadio
     type: Mobile App
     os: iOS
+  - regex: ^samsung-agent/
```

Comment:
Google seems to think this is a samsung smart tv.  Not sure what to call the OS, but I assume it's android-ish.